data.sql

```diff
@@ -44,3 +44,40 @@ UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Bob') WH
 UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Melody Pond') WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
 -- Dean Winchester owns Angemon and Boarmon.
 UPDATE animals SET owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester') WHERE name IN ('Angemon', 'Boarmon');
+
+/* Insert data into Vets table */
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('William Tatcher', 45, '04-23-2000');
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('Maisy Smith', 26, '01-17-2019');
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('Stephanie Mendez', 64, '05-04-1981');
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('Jack Harkness', 38, '04-23-2000');
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('William Tatcher', 45, '06-08-2008');
```

Comment:
- [ ] The `date_of_graduation` for `Jack Harness` is Jun 8th, 2008, kindly modify the entry.
- [ ] The last entry with the name `William Tatcher` is a duplicate entry, kindly remove it.

Reply:
Sure. Thank you